src/otel.js

```diff
@@ -0,0 +1,46 @@
+import { WebTracerProvider } from "@opentelemetry/sdk-trace-web";
+import { BatchSpanProcessor } from "@opentelemetry/sdk-trace-base";
+import { OTLPTraceExporter } from "@opentelemetry/exporter-trace-otlp-http";
+import { registerInstrumentations } from "@opentelemetry/instrumentation";
+import { FetchInstrumentation } from "@opentelemetry/instrumentation-fetch";
+import { DocumentLoadInstrumentation } from "@opentelemetry/instrumentation-document-load";
+import { SemanticResourceAttributes } from "@opentelemetry/semantic-conventions";
+import { resourceFromAttributes } from "@opentelemetry/resources";
+
+const resource = resourceFromAttributes({
+  [SemanticResourceAttributes.SERVICE_NAME]: "victory-center",
+});
+
+const exporter = new OTLPTraceExporter({
+  url: "http://localhost:4318/v1/traces",
+});
+
+const spanProcessor = new BatchSpanProcessor(exporter);
+
+const provider = new WebTracerProvider({
+  resource,
+  spanProcessors: [spanProcessor],
+});
```

Comment:
I guess it should be `provider.addSpanProcessor(spanProcessor);`
```suggestion
const provider = new WebTracerProvider({
  resource,
});

provider.addSpanProcessor(spanProcessor);
```